coalib/parsing/LineParser.py

```diff
@@ -78,6 +79,13 @@ def _parse(self, line):
                      [(section_override, key), ...], value, to_append (True if
                      append delimiter is found else False), comment
         """
+        if ((re.match('[^ ]#', line) or re.match('#[^ ]', line))
```

Comment:
the LineParser has a parameter comment_seperator so it's not necessarily #, we should obey that setting (or remove it)